osf/migrations/0186_registrationschema_registration_responses_jsonschema.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2019-09-08 19:42
+from __future__ import unicode_literals
+
+from django.db import migrations
+import osf.utils.datetime_aware_jsonfield
+from osf.utils.migrations import build_flattened_jsonschema
+
+
+def clear_registration_responses_jsonschema(state, schema):
+    RegistrationSchema = state.get_model('osf', 'registrationschema')
+    RegistrationSchema.objects.update(registration_responses_jsonschema={})
+
+
+def migrate_registration_responses_jsonschema(state, schema):
+    RegistrationSchema = state.get_model('osf', 'registrationschema')
+    for schema in RegistrationSchema.objects.all():
+        schema.registration_responses_jsonschema = build_flattened_jsonschema(schema)
```

Comment:
Stashes the jsonschema on each RegistrationSchema so we don't have to do extra db queries every time we need to validate `registration_responses`